docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -21,6 +21,36 @@ Playbook
 
  * ``hash_behaviour`` now affects inventory sources. If you have it set to ``merge``, the data you get from inventory might change and you will have to update playbooks accordingly. If you're using the default setting (``overwrite``), you will see no changes. Inventory was ignoring this setting.
 
+Tags on handlers
```

Comment:
Did this actually happen in 2.9 (since it's already branched) or does this change need to go into `porting_guide_2.10.rst`?

Reply:
No, it did not happen in 2.9 (the PR is still open). If we decide to go forward with this, the PR would have to be updated to move the porting guide entry from 2.9 to 2.10.